third-party/realdds/doc/notifications.md

```diff
@@ -0,0 +1,86 @@
+
+See also: [device](device.md), [initialization](initialization.md)
+
+
+# Notifications
+
+A topic is used to deliver [flexible](../include/realdds/topics/flexible/) messages from a server to its clients.
+
+> `<topic-root>` /notification
+
+These notifications can fall into many types, encompass almost any data, and happen at ANY time. Several are pre-defined and listed below.
+
+Notifications should always be a JSON (possibly CBOR) object (`{'x': true}` and not just `true`) and identified by an `id` field inside, e.g.:
+
+```JSON
+{
+    "id": "some-message-id",
+    "message": "this is a field value"
+}
+```
+
+Notification content is dependent on the `id`. Fields that are not recognized are ignored by the client. Notifications that are unrecognized are ignored. This allows the server to include additional content as needed.
+
+Messages are broadcast to all clients rather than to a specific one.
+
+
+#### Quality of Service
```

Comment:
This is a good section. Consider adding to discovery.md and other topic sections that will be added later

Reply:
Added to flexible, discovery, streaming